eval.py

```diff
@@ -1409,12 +1411,12 @@ def main():
         args.trained_model = SavePath.get_latest('weights/', cfg.name)
     elif is_valid_stub(args.trained_model):
         if args.engine == 'torch':
-            args.trained_model = get_checkpoint_from_stub(args.trained_model)
+            trained_model_weights = get_checkpoint_from_stub(args.trained_model)
```

Comment:
`args.trained_model` is a stub, which gets overwritten by a path to the `.ckpt` or `.pth` model. Overwriting the argument is bad practice, plus we need the stub later for benchmarking. This is the motivation behind the change.

Reply:
Nice!